explainer.md

```diff
@@ -180,70 +180,73 @@ Media Capabilities should offer a means of surfacing when different MediaDecodin
 
 ## HDR
 
-HDR support in browsers is nonexistent. The API is intended to enable high end media playback on the Web as soon as it becomes more mainstream so the platform does not lag behind the curve. This is also a great example of including more formats into the web and keeping the API extensible.
+The API is intended to enable high end media playback on the Web as soon as it becomes more mainstream so the platform does not lag behind the curve. This is also a great example of including more formats into the web and keeping the API extensible.
 
-### Screen capabilities
-
-Even if a device is able to decode HDR content, if the screen isn’t able to show this content appropriately, it might not be worth using HDR content for the website because of the higher bandwidth consumptions but also the rendering might be worse than a SDR optimised video.
-
-The following data can be used to define whether a screen is HDR-worthy:
-
-*   Colour gamut: HDR content requires a wider colour gamut than SDR content. Most screens use the sRGB colour gamut but p3 or BT.2020 are colour gamut that would usually be expected for HDR content.
-*   Colour/pixel depth: even if the screen has a wide colour gamut, the pixels need to be encoded in 30 bits (10 bits per colour component) instead of the usual 24 bits (8 bits per colour component). Otherwise, even if wider colour can be represented, precision would be lost.
-*   Brightness: because of darker blacks and brighter whites, a screen needs to have a large contrast ratio in order to be used for HDR content.
-
-#### Colour gamut
-
-The colour gamut of the screen could be exposed on the *Screen* interface. It is already part of the work in progress [CSS Media Queries 4](https://drafts.csswg.org/mediaqueries-4/#color-gamut) but because various information will have to be read from the *Screen* object for HDR content, it would make sense to have all of them grouped together.
+For HDR support detection, there are three main components whose capabilities need to be surfaced -- the decoder, renderer, and display. The decoder takes in an encoded stream and produces a decoded stream understood by the renderer, which in turn maps the stream's signals to those the display can properly output. Most of the time, the decoder and renderer are part of the UA while the display intuitively describes the physical output monitor, whether this be a computer monitor or TV. To match this natural modularity between the UA and the display, this API is compartmentalized into two (2) parts:
```

Comment:
Why does this section use "display" whereas the section that immediately follows use "screen" seemingly for the same concept? It is confusing as it stands. Can we define "screen" also if it is supposed to be different from "display"? Else use "screen" in this section?

Editorial: suggest replacing for simplicity:

"display intuitively describes the physical output monitor" with "display represents the physical output monitor"
"this API is compartmentalized into two (2) parts" with "this API is compartmentalized into two parts"